common/eq_packet_structs.h

```diff
@@ -1671,6 +1671,68 @@ struct GuildUpdate_Struct {
 	GuildsListEntry_Struct entry;
 };
 
+struct GuildMemberAdd_Struct {
+	/*000*/ uint32 guild_id;
+	/*004*/ uint32 unknown04;
+	/*008*/ uint32 unknown08;
+	/*012*/ uint32 unknown12;
+	/*016*/ uint32 level;
+	/*020*/ uint32 class_;
+	/*024*/ uint32 rank_;
+	/*028*/ uint32 spirit; //not confirmed single byte 0x000000FF
```

Comment:
Let's rename this field to indicate it's for the permission of showing the guild name

Reply:
Updated